src/Views/Textile/Step.elm

```diff
@@ -55,6 +56,7 @@ type alias Config msg =
     , updateAirTransportRatio : Maybe Split -> msg
     , updateDyeingMedium : DyeingMedium -> msg
     , updateEnnoblingHeatSource : Maybe HeatSource -> msg
+    , updateMaterialSpinning : Material -> Material.Spinning -> msg
```

Comment:
Nit: instinctively, I'd do the signature the other way around: `updateMaterialSpinning : Material.Spinning -> Material -> msg` so you can write stuff like `material |> updateMaterialSpinning spinning`

Reply:
We could, but in this specific case, we're using it the other way round: we already have the material, and we're updating the **spinning** for this material, see line 235. What do you think?

Reply:
Maybe rename as `updateSpinningMaterial` then? I agree that's a funny name 😅

Your call 🤷‍♂️

Reply:
We're updating the spinning of the material, so we're updating the material's spinning, so I guess the current name makes more sense.